CONTRIBUTING.md

```diff
@@ -61,7 +62,76 @@ about [how to report a new issue](#reporting-an-issue)!
     This is important to avoid the overlapping of work!
 
 1. If the issue is already in progress, feel free to ask questions or engage in conversation about
-    the implementation. Otherwise, continue on to [working on issues](#working-on-issues).
+    the implementation. Otherwise, review the repository license requirements
+    before starting work and continue on to [working on issues](#working-on-issues).
+
+### Reviewing license requirements
+
+To contribute to any CyberArk open source project, you must make sure you've met
+our contributor license requirements.
+
+To find out what the project you're interested in requires, refer to the
+`CONTRIBUTING.md` file in the repository.
+
+#### When the Repo Includes the CLA
+
+If the `CONTRIBUTING.md` document says you need to sign the CLA, for example:
```

Comment:
I think the wording here is too narrowly scoped. What if the repo doesn't "say that you need to sign a CLA" but says something like "CLA is required"?

Reply:
do you have a suggestion for how to fix it? I don't say the language on the CONTRIBUTING is _exactly_ "you need to sign", and I assume people can translate a CONTRIB that says "you are required to" to the "you need to" language here - but maybe I'm misunderstanding? 😬

Reply:
from a legal perspective, it may open things up to intentional unwanted avoidance of the CLA. Maybe we should say something like:
```
If the `CONTRIBUTING.md` document indicates that you need to sign the CLA, for example:
```